tools/loader/ImageClassifier.cpp

```diff
@@ -279,6 +284,25 @@ static int processAndPrintResultsImpl(Tensor *SMT,
   const size_t numLabels = SMT->dims()[1];
   std::vector<size_t> sliceOffset(SMT->dims().size(), 0);
 
+  // Parse provided, if any, category indices.
+  std::vector<signed> matchingIndices(inputImageFilenames.size(), -1);
+  if (!expectedMatchingIndices.empty()) {
+		llvm::SmallVector<llvm::StringRef, 1> parts;
+    llvm::StringRef(expectedMatchingIndices).split(parts, ',', -1, false);
```

Comment:
Can you please add a comment that describes what this line does?

Reply:
Fixed in the last commit